blocks/data-list/data-list.css

```diff
@@ -0,0 +1,38 @@
+.data-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.header-div, .row-div {
+    display: flex;
+    justify-content: space-between;
+    padding: 10px;
+    border-bottom: 1px solid #ddd;
+}
+
+.header-item, .row-item {
+    flex: 1;
+    text-align: center;
+}
+
+.header-div {
+    font-weight: bold;
+    background-color: #f4f4f4;
+}
+
+.pagination-controls {
+    margin-top: 10px;
+    display: block;
+    overflow: hidden;
+    float: right;
+}
+
+.pagination-controls button:first-of-type {
+    margin-right: 10px;
+}
+
+.content-container {
+    display: block;
+    overflow: hidden;
+    position: relative;
+}
```

Comment:
Always keep the block reference in the CSS otherwise it may lead of overrides.

Reply:
done